vtex-gocommerce/form-utils
# Add masked input to form

#### What does this PR do? 

I need masked inputs to do the billing form on vtex local wizard.

Adds a Form.MaskedInput component that accepts mask patterns like "999.99", or array mask patterns like ['/\d/', '/\d/', '/\d/', '.', '/\d/', '/\d/'] (which can be more customized).

Currently using MaskedInput from 'react-text-mask' lib.

#### How to test it? 

[Running workpace](https://dev--gc-wmr3721.mygocommerce.com/admin/wizard/vtex-local)
Please go to page 2 to see a CPF input and check the browser console to see the form values printed out.

#### Describe alternatives you've considered, if any.

Use react-imask to mask the input but it didn't work 😭
Use gocommerce styleguide input which is already using 'react-text-mask' to mask the input.

We should change this to another lib later since 'react-text-mask' is not being supported anymore.

#### Related to / Depends on
